deploy/scripts/src/spot/requestGenerator/spotReqGen.ts

```diff
@@ -1,6 +1,8 @@
 import { type SpotRequest } from '../../spot/requestGenerator/spotReqFormat'
 import { uuidv4 } from '../../common/utils/jslib/index'
 
+const audClientID = uuidv4()
+
 export function generateRequest (currTime: string): SpotRequest {
```

Comment:
```suggestion
export function generateRequest (currTime: string): SpotRequest {
  const audClientID = uuidv4()
```
Move the variable declaration inside the function so there is a new UUID for every iteration

Reply:
Done